libraries/Adapters/Microsoft.Bot.Builder.Adapters.Webex/WebexAdapter.cs

```diff
@@ -300,7 +161,7 @@ public async Task ContinueConversationAsync(ConversationReference reference, Bot
         /// <param name="bot">A bot with logic function in the form.</param>
         /// <param name="cancellationToken">A cancellation token for the task.</param>
         /// <returns>A <see cref="Task"/> representing the asynchronous operation.</returns>
-        public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot bot, CancellationToken cancellationToken = default)
+        public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot bot, CancellationToken cancellationToken)
```

Comment:
Shouldn't we use `CancellationToken cancellationToken = default`

Reply:
Gabo asked to keep the CancellationToken as mandatory except in cases where there are other optional parameters before.